prisma/schema.prisma

```diff
@@ -94,9 +85,9 @@ model Post {
 model Comment {
   id        Int      @id @default(autoincrement())
   post      Post     @relation(fields: [postId], references: [id])
-  postId    Int      @unique
+  postId    Int      
   profile   Profile  @relation(fields: [profileId], references: [id])
-  profileId Int      @unique
+  profileId Int      
```

Comment:
removing unique here does make sense

Reply:
How will the profile id not be unique? I dont get how two profiles can have the same id?

Reply:
see comment above - I reckon this should be unique...

Reply:
this was on the comment model; if it were unique, users could only have 1 comment!

Reply:
Yeah, I've had problems with unique constraints with the ratings model. It wouldn't work if it was unique